test/functional/test_framework/wallet.py

```diff
@@ -188,6 +188,41 @@ def send_to(self, *, from_node, scriptPubKey, amount, fee=1000):
         txid = self.sendrawtransaction(from_node=from_node, tx_hex=tx.serialize().hex())
         return txid, 1
 
+    def send_self_transfer_multi(self, from_node, utxos, **kwargs):
+        """
+        Create and send a transaction that spends the given UTXOs and creates a
+        certain number of outputs with equal amounts.
+
+        Returns a list of newly created UTXOs, ordered by vout index.
+        """
+        tx = self.create_self_transfer_multi(from_node, utxos, **kwargs)
+        txid = self.sendrawtransaction(from_node=from_node, tx_hex=tx.serialize().hex())
+        return [self.get_utxo(txid=txid, vout=vout) for vout in range(len(tx.vout))]
```

Comment:
nit: Maybe return a dict similar to `send_self_transfer`?

```suggestion
        return {"new_utxo":[self.get_utxo(txid=txid, vout=vout) for vout in range(len(tx.vout))], "tx": tx}
```